doc/dsl/typebuilding.md

```diff
@@ -16,9 +16,13 @@ game_object my_object {
 }
 ```
 
-Diese Form der Entitätsdefinition erfordert eine Repräsentation der Komponententypen (welche als Java-Klassen / Java-Records im ECS definiert sind) im DSL Typsystem und eine Verbindung des DSL Typen mit der Java-Klasse, dessen DSL-Equivalent er ist.
+Diese Form der Entitätsdefinition erfordert eine Repräsentation der Komponententypen (welche als
+Java-Klassen / Java-Records im ECS definiert sind) im DSL Typsystem und eine Verbindung des
```

Comment:
Records sind eine spezielle Form von Klassen. Würde ich hier eher nicht unterscheiden?

Reply:
Hab ich rausgenommen, die einzige Unterscheidung ist jetzt in dem Einschränkungs-Kapitel